src/falco/test.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+echo "> Prepare test data"
+
+set -eo pipefail
+
+# We use data from this repo: https://github.com/hartwigmedical/testData
+echo ">> Fetching and preparing test data"
+fastq1="https://github.com/hartwigmedical/testdata/raw/master/100k_reads_hiseq/TESTX/TESTX_H7YRLADXX_S1_L001_R1_001.fastq.gz"
+fastq2="https://github.com/hartwigmedical/testdata/raw/master/100k_reads_hiseq/TESTX/TESTX_H7YRLADXX_S1_L001_R2_001.fastq.gz"
+test_data_dir="test_data"
+
+mkdir $test_data_dir
+wget -q $fastq1 -O $test_data_dir/R1.fastq.gz
+wget -q $fastq2 -O $test_data_dir/R2.fastq.gz
```

Comment:
I think other components just copied the data into a folder `test_data`. I think here we should do the same to be consistent?

Reply:
Good point. I did something similar like this with the bcl_convert component because actually storing a complete run in the `test_data` folder would be too much. In this case, the fastq files I'm using are _only_ 2.5MB but it adds up if more components start to do this.

My primary concern is that the test runs standalone?

Reply:
True, I would also like to avoid the situation where the clone of the repo starts to take a lot of time because it is full of test data....